autodiffcc/root.py

```diff
@@ -24,13 +24,31 @@ def _norm(vector):
 
 
 def _newton_raphson(function, values, threshold, max_iter):
-    """#TODO
+    """Returns a root found starting from values or raised Exception if none are found
```

Comment:
@aspiride when you review my PR #57 you might need to deconflict a merge conflict as I also wrote Newton-Raphson docstring.